crypto/doc/CHANGELOG_cc310_bl.rst

```diff
@@ -17,15 +17,15 @@ Initial release.
 Added
 =====
 
-Added the following crypto libraries for cc310 bootloader nRF52840 and nRF9160.
+Added the following crypto libraries for cc310 bootloader nRF52840 and nRF91.
 
 .. note::
-   * The cc310 libraries require the CryptoCell 310 HW accelerator, available on nRF52840 and nRF9160 devices.
+   * The cc310 libraries require the CryptoCell 310 HW accelerator, available on nRF52840 and nRF91 devices.
    * short-wchar: These libraries are compiled with a wchar_t size of 16 bits.
    * no-interrupts: These libraries use blocking waits for long-running cryptography calculations.
 
 
-* nrf_cc310_bl, nrf9160 variants:
+* nrf_cc310_bl, nrf91 variants:
```

Comment:
```suggestion
* nrf_cc310_bl, nRF91 Series variants:
```

Unless these are written to display exactly as something in code, but then they should be styled some way.